numba_typing/tests/test_type_annotations.py

```diff
@@ -0,0 +1,46 @@
+import unittest
+import type_annotations
+from typing import Union, Dict, List
+
+
+class TestTypeAnnotations(unittest.TestCase):
+
+    def test_get_func_annotations_exception(self):
+
+        def foo(a: int, b, c: str = "string"):
+            pass
+        self.assertRaises(SyntaxError,  type_annotations.get_func_annotations, foo)
+
+    def test_get_cls_annotations(self):
+        class MyClass(object):
+            x: int = 3
+            y: str = "string"
+
+            def __init__(self, x: str, y: int):
```

Comment:
That's a very strange test. Does it makes intentionally what annotation of class members differs from annotation in __init__?

That's an interesting case, but in this case it causes only confusion